jupyter/utils/addons/apply.sh

```diff
@@ -31,6 +31,12 @@ fi
 
 curl -o "$static_dir/pf.css" "$pf_url"
 
+# Patternfly implemented some changes, setting the default height on iframe objects to "auto", which
+# excluded the default 800px height of Tensorboard's IFrame object. This stylesheet will simply search
+# for all IFrames that contain `tensorboard` on its URL and add a default height of 800px to it, and only
+# on it.
+echo "iframe[id*=\"tensorboard\"] { height: 800px !important }" >> "$static_dir/pf.css"
```

Comment:
Nice hack, but being able to have patternfly without css reset would be much better. Who knows what else is patternfly breaking and we just don't know about it. Customers may have their own extensions like tensorboard that are broken by different patternfly css rules.

Reply:
My main concern, at least right now, is if taking out the CSS reset out would break other layout or other implementations. I would like to have this fixed right now WITH the reset and, if possible, we could try later implementing the no-reset version and checking if everything still works as expected with proper time for testing.

Reply:
@caponetto ^^^ you were right saying on slack that adding the whole patternfly on top of jupyterlab was not the most fortunate move, it makes things harder down the line

Reply:
> My main concern, at least right now, is if taking out the CSS reset out would break other layout or other implementations.

@daniellutz AFAIK, the only reason we added this patternfly thing was for the initial loading spinner at the beginning of the workbench IDE load. So unless this thing/feature breaks, we don't need to care about anything else.

Reply:
exactly as @jstourac says, that's why I'm calling out caponetto's prescient words